completions/openssl

```diff
@@ -110,6 +111,9 @@ _comp_cmd_openssl()
                 dgst | req | x509)
                     _comp_compgen -a -i openssl digests "$1"
                     ;;
+                enc)
+                    _comp_compgen -- -W "$("$1" enc -list)"
```

Comment:
```suggestion
                    _comp_compgen_split -- "$("$1" enc -list)"
```

The argument to `-W` is subject to expansion, so we prepared `_comp_compgen_split` to prevent the output of command substitutions from undergoing shell expansions. Even when we know that the present specific call of command substitution doesn't produce outputs looking like shell expansions, it is not obvious just by looking at this line. We might write `-W '"$(...)"'` instead, but shellcheck wouldn't be applied that way.

Reply:
Done